src/main/java/dk/kb/util/webservice/OpenApiResource.java

```diff
@@ -195,7 +197,27 @@ private static String getReplacementForMatch(String yPath) {
         if (config == null){
             throw new IllegalStateException("Config must be initialized before using the class. See JavaDoc for OpenApiResource for further details.");
         }
-        return config.get(yPath).toString();
+
+        List<Object> result = config.getMultiple(yPath);
+
+        if (result.isEmpty()){
+            // I am not quite sure what we should return here. Is it too harsh to throw an exception? If not, which would be correct? Illegal Argument?
+            log.error("No entry has been found for yPath: '{}'.", yPath);
+            return yPath;
```

Comment:
`log.error` signals a catastrophic error, which I think is quite fair here: If there's a missing config then there's a big problem. So keep it and yes, throw an exception. `IllegalArgument` seems like a fine choice.